### PR TITLE
Feature/context file

### PR DESCRIPTION
**Pull request by duckworth**
Adds a default empty context.json file which can be used to add additional mock data to the context object passed in to the lambda function for the run command. Adds a -x --contextFile param to override the contextFile used.

### CODE REVIEW

**Comment by ubiquitousthey** [comment]
This is great thing to add

**Comment by motdotla** [comment]
Thanks for this @duckworth. I'll take a look at this, this weekend.

**Comment by DeviaVir** [comment]
@duckworth I realize there is no `.editorconfig` attached to this project yet, but I'd appreciate it if you could update your changes to have the indenting as it was (2 spaces). It makes it also a lot harder to read your changes!

**Comment by DeviaVir** [comment]
ping @duckworth, do you still wish to own these changes?

**Comment by duckworth** [comment]
ok, the formatting should be resolved.

File: README.md [hunk 1]
```diff
@@ -64,9 +64,10 @@ $ node-lambda run --help
 
   Options:
 
-    -h, --help                     output usage information
-    -h, --handler [index.handler]  Lambda Handler {index.handler}
-    -j, --eventFile [event.json]   Event JSON File
+    -h, --help                          output usage information
+    -h, --handler [index.handler]       Lambda Handler {index.handler}
+    -j, --eventFile [event.json]        Event JSON File
+    -x, --contextFile [context.json]    Context JSON file
```
Comment (DeviaVir) [inline README.md]: I think using `-x` for excludes makes a bit more sense:
https://github.com/motdotla/node-lambda/pull/48
Perhaps using `-c` makes sense here?

**Comment by DeviaVir** [comment]
@duckworth thanks! I agree with the changes, unfortunately your branch is not mergeable due to changes upstream on master creating conflicts. Could you resolve that?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-lambda",
-  "version": "0.7.1",
+  "version": "0.7.2",
```
Comment (DeviaVir) [inline package.json]: The idea is to bump this up with https://github.com/motdotla/node-lambda/pull/53 so there's no need for this change here.

File: test/main.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ describe('node-lambda', function () {
   });
 
   it('version should be set', function () {
-    assert.equal(lambda.version, '0.7.1');
+    assert.equal(lambda.version, '0.7.2');
```
Comment (DeviaVir) [inline test/main.js]: https://github.com/motdotla/node-lambda/pull/40/files#r59985448

**Comment by duckworth** [comment]
Should I switch to use -c ? I didn't use that originally as that was already used on the deploy command. It appears that has been removed.

`deploy -c, --mode [event]                Lambda Mode`

**Comment by DeviaVir** [comment]
@duckworth I removed my original comment after noticing that the PR I referenced used a different run command on conjunction with the `-x` so that can be ignored!

**Comment by DeviaVir** [comment]
@duckworth let me know if you need any help making your branch compatible (mergeable) with the current upstream/master.

**Comment by DeviaVir** [comment]
Deprecated by #82

**Pull request closed without merging**